tests/test_factory.py

```diff
@@ -460,6 +460,15 @@ def test_random_pyfloat(self):
         with pytest.raises(ValueError):
             provider.pyfloat(left_digits=0, right_digits=0)
 
+    def test_negative_pyfloat(self):
+        # tests for https://github.com/joke2k/faker/issues/813
+        from faker.providers.python import Provider
+        provider = Provider(self.generator)
+
+        assert any(provider.pyfloat(left_digits=0, positive=False) < 0 for _ in range(100))
```

Comment:
can you make these tests deterministic by using [`Provider.seed_instance()`](https://github.com/joke2k/faker#seeding-the-generator)?

Reply:
I can and will. I omitted that because the probability of failure is somewhere around 2^-99.